src/api/src/endpoints/account/request_account_deletion.rs

```diff
@@ -0,0 +1,44 @@
+use actix_web::{web, HttpResponse};
+use chrono::Utc;
+use serde::Deserialize;
+use ticketland_core::error::Error;
+// use ticketland_data::models::account::Account;
+use api_helpers::middleware::auth::AuthData;
+use crate::utils::store::Store;
+
+#[derive(Deserialize)]
+pub struct QueryString {
+  delete_request: bool,
```

Comment:
We should avoid using a boolean flags. This is one of the first things I tried to teach you guys. It's better to repeat the code twice rather than adding a confusing boolean flag.